src/core/mac/mac_frame.hpp

```diff
@@ -733,7 +733,8 @@ class Frame : public otRadioFrame
     otError SetCommandId(uint8_t aCommandId);
 
     /**
-     * This method indicates whether the frame is a MAC Data Request command (data poll)
+     * This method indicates whether the frame is a MAC Data Request command (data poll).
+     * For 802.15.4-2015 and above frame, the frame should be already decrypted.
```

Comment:
```suggestion
     * This method indicates whether the frame is a MAC Data Request command (data poll).
     
     * For 802.15.4-2015 and above frame, the frame should be already decrypted.
```
nit: As an [example](https://github.com/openthread/openthread/blob/master/src/core/mac/mac.hpp#L234)